packages/request-manager/src/interceptor.ts

```diff
@@ -130,9 +132,9 @@ const interceptHttps = (interceptorOptions: InterceptorOptions) => {
                 delete overloadedOptions.headers['Proxy-Authorization'];
             }
             // Requests to localhost should not use the proxy.
-            const shouldIntercept = overloadedOptionsUrl.hostname !== '127.0.0.1';
+            const isNotLocalhost = !isLocalhost(overloadedOptionsUrl.hostname);
```

Comment:
shouldnt we have this block of code, and the block above (http.request) done as one function? its like 30+ duplicated lines

Reply:
thats right, `isNotLocalhost` util would make sense here

Reply:
Yes, I created issue https://github.com/trezor/trezor-suite/issues/6674 to address that.

Reply:
ok, merging.